gulpfile.js

```diff
@@ -113,6 +125,50 @@ gulp.task('typings', (done) => {
   done();
 });
 
+gulp.task('variants', (done) => {
+  colorSchemes.map(({transformType, formatType}) =>
+    gulp
+      .src('tokens/themes/*.yml')
+      .pipe($.rename(addPrefix))
+      .pipe(
+        $.theo({
+          transform: {type: transformType},
+          format: {type: formatType},
+        }),
+      )
+      .pipe(
+        $.theo({
+          transform: {type: 'web/js'},
+          format: {type: 'json'},
```

Comment:
Replace this set with a map over `colorSystemFormats` for converting these files into all of the new formats we want.